frontend/app/model/FeatureChoice.scala

```diff
@@ -0,0 +1,42 @@
+package model
+
+trait FeatureChoice {
+  val id: String
+  val label: String
+  val codes: Set[String]
+}
+
+object FeatureChoice {
+  val all: Seq[FeatureChoice] = Seq(
+    Events,
+    Books
+  )
+
+  val codes = all.foldLeft(Set.empty[String]) {
+    case (_codes, choice) => _codes ++ choice.codes
+  }
+
+  def fromString(id: String): Option[FeatureChoice] = id match {
+    case Books.id => Some(Books)
+    case Events.id => Some(Events)
+    case BooksAndEvents.id => Some(BooksAndEvents)
+    case _ => None
+  }
+}
+
+case object Books extends FeatureChoice {
+  override val id    = "Books"
+  override val label = "4 free books"
+  override val codes = Set(id)
+}
+case object Events extends FeatureChoice {
+  override val id    = "Events"
+  override val label = "6 free events"
+  override val codes = Set(id)
+}
+
+case object BooksAndEvents extends FeatureChoice {
```

Comment:
Is it necessary to create a special BooksAndEvents feature?! Surely this should be represented as a set that can contain either `Books` or `Events` or both?